pkg/asset/asset.go

```diff
@@ -50,6 +50,9 @@ type Config struct {
 	CACert          *x509.Certificate
 	CAPrivKey       *rsa.PrivateKey
 	AltNames        *tlsutil.AltNames
+	PodCIDR         string
+	ServiceCIDR     string
+	DNSServiceIP    string
```

Comment:
We can. Render can parse strings to ipnets and back to strings on the other side.